dashboard/app/models/level_source_image.rb

```diff
@@ -71,7 +66,22 @@ def upload_framed_image(image)
     upload_image(s3_framed_filename, framed_image)
   end
 
-  def self.hashify_filename(plain)
+  def delete_image_or_framed_image
+    AWS::S3.delete_from_bucket(S3_BUCKET, s3_filename)
+    AWS::S3.delete_from_bucket(S3_BUCKET, s3_framed_filename)
+  end
+
+  private
+
+  # @return [String] The filename for the image frame.
+  def framed_image_filename
+    if ['anna', 'elsa'].include? level_source.level.try(:skin)
+      return "app/assets/images/blank_sharing_drawing_#{level_source.level.skin}.png"
+    end
+    'app/assets/images/blank_sharing_drawing.png'
+  end
```

Comment:
i wonder if this would better live on the Level model? i.e. we do something like `level_source.level.framed_image_filename`, and it then looks at the skin and decides how to behave. Not sure if I feel strongly one way or the other.